flatten_json_test.go

```diff
@@ -123,7 +123,7 @@ func TestMinimal(t *testing.T) {
 }
 
 func testTrackerSelection(fj Flattener, tracker NameTracker, label string, filename string, wantedPaths []string, wantedVals []string, t *testing.T) {
-	event, err := ioutil.ReadFile(filename)
+	event, err := os.ReadFile(filename)
```

Comment:
Fixing deprecation warning - 
```
flatten_json_test.go:4:2: SA1019: "io/ioutil" has been deprecated since Go 1.16: As of Go 1.16, the same functionality is now provided by package io or package os, and those implementations should be preferred in new code. See the specific function documentation for details. (staticcheck)
        "io/ioutil"
```